edi_purchase_oca/views/res_partner_view.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="view_partner_form_inherit" model="ir.ui.view">
+        <field name="name">res.partner.form.inherit.sales_purchases</field>
+        <field name="model">res.partner</field>
+        <field name="inherit_id" ref="base.view_partner_form" />
+        <field name="arch" type="xml">
+            <xpath expr="//page[@name='sales_purchases']" position="inside">
```

Comment:
better in its own tab EDI no?

Reply:
Because it is in the depend relationship of **`edi_purchase_oca`**. The **`EDI tab`** is not created yet. So I will create for it an **`EDI Configurations`** tab to contain the relationships between **partner**, **model**, **edi.configuration**